### PR TITLE
Remove Quick Start

### PR DESCRIPTION
**Pull request by kean**
This PR removes Quick Start. This will allow us to safely make the navigation changes for iPad on iOS 18, and also close the existing 42+ bugs related to Quick Start.

The plan is to replace it in the future with a much simpler system similar to a wp.com Launchpad for new sites.

## To test:

This change affects a lot of screens but mainly the following areas:

- Site creation flow
- Login flow with existing account (especially "enable notifications" prompt)

## Regression Notes
1. Potential unintended areas of impact

2. What I did to test those areas of impact (or what existing automated tests I relied on)

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:

- [ ] I have completed the Regression Notes.
- [ ] I have considered adding unit tests for my changes.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

Testing checklist:
- [ ] WordPress.com sites and self-hosted Jetpack sites.
- [ ] Portrait and landscape orientations.
- [ ] Light and dark modes.
- [ ] Fonts: Larger, smaller and bold text.
- [ ] High contrast.
- [ ] VoiceOver.
- [ ] Languages with large words or with letters/accents not frequently used in English.
- [ ] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [ ] iPhone and iPad. 
- [ ] Multi-tasking: Split view and Slide over. (iPad)

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">View files have been modified, but no screenshot or video is included in the pull request. Consider adding some for clarity.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 500 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by kean** [comment]
I'll need some help testing it, folks, @jkmassel , @crazytonyli .
I ran a smoke test, but I'm wrapping it up for today and will continue tomorrow.

**Comment by crazytonyli** [comment]
@kean You may need to rebase this PR, because it looks like changes in https://github.com/wordpress-mobile/WordPress-iOS/pull/23470 got in here.

**Comment by kean** [comment]
Yeah, it was because of the squash-merge. Updated.

File: WordPress/Classes/Models/QuickStartTourState.swift [hunk 1]
```diff
@@ -1,3 +1,6 @@
+import CoreData
+
+@available(*, deprecated, message: "No longer used")
```
Comment (crazytonyli) [inline WordPress/Classes/Models/QuickStartTourState.swift:3]: `obsolete` is probably more appropriate than `deprecated` here.

**Review by crazytonyli — APPROVED** [review]

File: WordPress/Classes/Models/QuickStartTourState.swift (hunk 1 above)
Comment (kean) [reply WordPress/Classes/Models/QuickStartTourState.swift:3]: I'll remove these in the upcoming PRs.

**Comment by kean** [comment]
FYI I pushed a quick update to UI tests to skip the "Enable Notifications" screen automatically and to disable all tips from TipKit.

**Pull request merged**